project/OcsBundle.scala

```diff
@@ -355,7 +355,7 @@ trait OcsBundle {
   lazy val bundle_edu_gemini_ags = 
     project.in(file("bundle/edu.gemini.ags")).dependsOn(
       bundle_edu_gemini_shared_skyobject,
-      bundle_edu_gemini_pot,
+      bundle_edu_gemini_pot          % "test->test;compile->compile",
```

Comment:
I wonder what's the difference, I type `reload` but I see the same with a restart, do you think some old caches are affecting this? I should try the `reload return` you mentioned a while ago